tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/Contact-us.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Contact US and Support</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css" />
+  </head>
+  <body id="contact">
+    <div id="contact-us-page">
+      <h1>Contact US and Support</h1>
+      <p>Address.:601, Uzair Tower, Next to Zoya Complex, ********.</p>
+      <p>Mob.No.:+91 887999****</p>
+      <p>E-mail.:tauqueerrkhan@gmail.com</p>
```

Comment:
try  using tel: inside the href attribute @Tauqeer-Ahmed-99 
 
try using mailto: inside the href attribute @Tauqeer-Ahmed-99

Reply:
done changes @fayez-baig